packages/rocketchat-federation/package.js

```diff
@@ -0,0 +1,40 @@
+Package.describe({
+	name: 'rocketchat:federation',
+	version: '0.0.1',
+	summary: 'RocketChat support for federating with other RocketChat servers',
+	git: '',
+});
+
+Package.onUse(function(api) {
+	api.use([
+		'ecmascript',
+		'rocketchat:api',
+		'rocketchat:lib',
+		'rocketchat:reactions',
+		'rocketchat:models',
+		'rocketchat:settings',
+	]);
+
+	api.use('accounts-base', 'server');
+	api.use('accounts-password', 'server');
+
+	// Add models
+	api.addFiles('server/models/FederationDNSCache.js', 'server');
+	api.addFiles('server/models/FederationEvents.js', 'server');
+	api.addFiles('server/models/FederationKeys.js', 'server');
+
+	// Add methods
+	api.addFiles('server/methods/index.js', 'server');
+
+	// Add core files
+	api.addFiles('server/federation.js', 'server');
+	api.addFiles('server/federation-settings.js', 'server');
+
+	api.mainModule('server/index.js', 'server');
```

Comment:
as we decided moving to `api.mainModule`, we should not mix it with `api.addFiles` ..

so please remove all `api.addFiles` and make sure the files are imported.

Reply:
Oh ok, got it.